metadata_mapper/mappers/mapper.py

```diff
@@ -11,7 +11,8 @@
 from datetime import date
 
 import settings
-from .validator import Validator, ValidationErrors
+from validator.validator import Validator
```

Comment:
I think it makes sense to assume that we are using the `mapper.py` module as an import instead of running it directly as a script, which case we can change this to:

`from validator import validator`